includes/class-google-fonts.php

```
@@ -90,7 +90,10 @@ public function __construct( array $options ) {
 		 */
 		public function process_style_tag( $tag, $handle, $href ) {
 			if ( is_user_logged_in() ) {
-				return $tag;
+				$logged_in_cache_enabled = ! empty( $this->options['cache_settings']['enableLoggedInCache'] ?? false );
```

PHPMD (2.15.0)

[warning] Avoid excessively long variable names like $logged_in_cache_enabled. Keep variable name length under 20. (undefined)

(LongVariable)